benchmarks/_sources.py

```diff
@@ -490,34 +451,13 @@ def _get_object_url(self, idx=0):
     def download_source_if_not_exists(self):
         for idx, p in enumerate(self.source_paths):
             path = pathlib.Path(p)
-            if self._if_path_does_not_exist_or_not_expected_file_size(path):
+            if not path.exists():
                 path.parent.mkdir(parents=True, exist_ok=True)
                 source = self.store.get("source")
                 if not source:
                     source = self._get_object_url(idx)
                 r = requests.get(source)
                 open(path, "wb").write(r.content)
-                self._assert_expected_file_size(path)
-
-    def _if_path_does_not_exist_or_not_expected_file_size(self, path):
-        expected = bytes_fmt(EXPECTED_SIZES.get(path.name))
-        return not path.exists() or bytes_fmt(os.path.getsize(path)) != expected
-
-    def _assert_expected_file_size(self, path):
-        expected = EXPECTED_SIZES.get(path.name)
-        expected_formatted = bytes_fmt(expected)
-        actual = os.path.getsize(path)
-        actual_formatted = bytes_fmt(actual)
-        debug = [
-            path.name,
-            expected_formatted,
-            actual_formatted,
-            expected,
-            actual,
-        ]
-        skip = ["data.parquet", "data.feather"]  # TODO
-        if path.name not in skip:
-            assert expected_formatted == actual_formatted, debug
```

Comment:
I'm fine deferring this type of checking for now. Ultimately it should be part of datalogistik anyway. Would you mind making an issue at https://github.com/conbench/datalogistik referencing this code snippet so we can follow back up on that if we need to?